api/ssi-bridge/src/services/tests/configuration-service.test.ts

```diff
@@ -8,32 +8,54 @@ describe('test configuration service', () => {
 	let loggerSpy: jest.SpyInstance;
 	beforeEach(() => {
 		loggerSpy = jest.spyOn(LoggerMock, 'error');
+		ConfigurationService.instance = undefined;
 	});
 
 	it('should crash since env variables are missing', async () => {
-		const start = () => ConfigurationService.getInstance(LoggerMock);
-		expect(start).toThrowError('Server secret must to have a length of 32!');
+		const mockExit = jest.spyOn(process, 'exit').mockImplementation(() => {
+			return undefined as never;
+		});
```

Comment:
cant this just be
```
...mockImplementation(() => null)
```
?

Reply:
Sadly not possible since process.exit() returns a never. Got the code snippet from this [comment](https://stackoverflow.com/questions/46148169/stubbing-process-exit-with-jest#comment105698413_51448240) 
This is the resulting error when using your proposal. 
`Type 'any' is not assignable to type 'never'.`

Reply:
ok could you please move the mockexit to the beforeeach and use a global variable as `let mockExit` so we don't have it in every test case